cicd/index.adoc

```diff
@@ -24,18 +33,32 @@ For more information, see link:https://docs.openshift.com/builds/latest/about/ov
 * Builds using `BuildConfig` objects is a declarative build process to create cloud-native apps. You can define the build process in a YAML file that you use to create a `BuildConfig` object. This definition includes attributes such as build triggers, input parameters, and source code. When deployed, the `BuildConfig` object builds a runnable image and pushes the image to a container image registry. With the `BuildConfig` object, you can create a Docker, Source-to-image (S2I), or custom build.
 +
 For more information, see xref:../cicd/builds/understanding-image-builds.adoc#understanding-image-builds[Understanding image builds].
+endif::openshift-dedicated,openshift-rosa[]
 
+// This is the same para as above, but without the unordered list.
+ifdef::openshift-dedicated,openshift-rosa[]
+Builds using `BuildConfig` objects is a declarative build process to create cloud-native apps. You can define the build process in a YAML file that you use to create a `BuildConfig` object. This definition includes attributes such as build triggers, input parameters, and source code. When deployed, the `BuildConfig` object builds a runnable image and pushes the image to a container image registry. With the `BuildConfig` object, you can create a Docker, Source-to-image (S2I), or custom build.
```

Comment:
"Builds... is a declarative build process" sounds grammatically incorrect, maybe you can fix it? But I do understand it's pre-existing text